src/sys/node/logger/index.ts

```diff
@@ -0,0 +1,100 @@
+import { Logger } from '@stencil/core/declarations';
```

Comment:
As far as I can tell, with the exception of imports, this file is a copy of `src/sys/node/node-logger.ts` (which we delete in this PR). Looking at this file's history (`git log -p -- src/sys/node/logger/index.ts`), it looks like this is a new file rather than a move:
```
diff --git a/src/sys/node/logger/index.ts b/src/sys/node/logger/index.ts
new file mode 100644
index 000000000..dce445409
--- /dev/null
+++ b/src/sys/node/logger/index.ts
```

Can you do me a favor a use `git mv` to move/rename the old file instead? That way, we can preserve git history between the two.

Reply:
good idea - I made that update, so it's now properly noted as a move rather than a delete + creation